plugins/modules/zos_mount.py

```diff
@@ -1109,6 +1116,7 @@ def main():
                 required=False,
             ),
             automove_list=dict(type="str", required=False),
+            tmphlq=dict(type='str', required=False, default=None),
```

Comment:
There is confusion... what I thought I saw was 'tmphlq' is a string/parameter, and tmp_hlq is a dictionary. As I look more closely, I'm seeing them intermixed. I'll try to put comment notes where I see tmphlq as a dictionary.